src/components/LogView/LogLineText.js

```diff
@@ -38,6 +78,14 @@ export default class LogLineText extends React.PureComponent<Props> {
     this.lineRef = element;
   };
 
+  containsValidJSON(str: string) {
```

Comment:
This function is unused

Reply:
Wow, my linter is really missing a lot! Thanks for finding all of these.